lib/dotcom_web/components/trip_planner/input_form.ex

```diff
@@ -111,6 +104,32 @@ defmodule DotcomWeb.Components.TripPlanner.InputForm do
     """
   end
 
+  defp location_search_box(assigns) do
+    assigns =
+      assigns
+      |> assign(:location_keys, InputForm.Location.fields())
+
+    ~H"""
+    <fieldset class="mb-sm -mt-md" id={"#{@name}-wrapper"}>
+      <legend class="text-charcoal-40 m-0 py-sm">{Phoenix.Naming.humanize(@field.field)}</legend>
+      <.algolia_autocomplete config_type="trip-planner" placeholder="Enter a location" id={@name}>
```

Comment:
```suggestion
      <.algolia_autocomplete config_type="trip-planner" placeholder={@placeholder} id={@name}>
```

We lost [this change](https://github.com/mbta/dotcom/pull/2300/files#diff-6c63a22d4645df0238c7ebc4e7754b74177b5b8ff240cfde7212fadb5a74dadfR39), which is helpful for the detached search box experience.